rm/rm.c

```diff
@@ -1295,9 +1308,10 @@ static bool rev_session_clink_cb(struct m0_clink *link)
 			rem_rev_sess_clink, &rem_bob);
 	resource = remote->rem_resource;
 
-	if (m0_rpc_session_status(remote->rem_session) != 0)
-		/* The connection has failed. Stop further processing. */
+	if (m0_rpc_session_status(remote->rem_session) != 0) {
+		M0_LEAVE("remote rev_session status is not good");
```

Comment:
Is it necessary to have the "status" in the message?

Reply:
Yes, it might be helpful, let me fix that...